aspnetcore/signalr/client-features.md

```diff
@@ -0,0 +1,28 @@
+---
+title: SignalR Client Features
+author: bradygaster
+description: Features as supported by the various ASP.NET Core SignalR clients.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: bradyg
+ms.custom: mvc
+ms.date: 09/18/2019
+uid: signalr/client-features
+---
+# ASP.NET Core SignalR client features
+
+## Feature distribution
+
+The table below illustrates the matrix of features and their support across the clients that offer real-time support to various platforms. 
+
+| Feature | .NET Core | JavaScript | Java |
+| ---- | :-: | :-: | :-: |
+| [Server-to-client Streaming](xref:signalr/streaming)          |✔|✔| |
```

Comment:
```suggestion
| [Server-to-client Streaming](xref:signalr/streaming)          |✔|✔|✔|
```

Reply:
Do we even need a table then?

Reply:
Nvm. I guess we keep it for auto-reconnect.

Reply:
Also we'll probably be adding more rows

Reply:
also think about things like long polling, sse - how are those supported across all the clients. **absolutely** just isn't about "new features" as much as it is answering the perpetual "what can i do where" question.